openfisca_country_template/variables/demographics.py

```diff
@@ -21,14 +30,10 @@ class age(Variable):
     # A person's age is computed according to its birth date.
     def formula(person, period, parameters):
         birth = person('birth', period)
-        return (datetime64(period.date) - birth).astype('timedelta64[Y]')
+        birth_year = birth.astype('datetime64[Y]').astype(int) + 1970
+        birth_month = birth.astype('datetime64[M]').astype(int) % 12 + 1
+        birth_day = (birth - birth.astype('datetime64[M]') + 1).astype(int)
```

Comment:
> Frankly ugly, but we don't have better yet

Reply:
I'd _really_ like that a time helper library is researched and vetted for.

Reply:
We started to ask around, and there are candidates (Maya, Arrow, etc.). Though in this specific case, we need to handle _vectors_ of dates, which will be a clear limitation.
If we were to use a library, we should also probably also use it to handle periods. So IMO this is related to https://github.com/openfisca/openfisca-core/issues/670.